### DISCUSSION TITLE
how to start WireSockUI (0.2.6) as non admin?

### DISCUSSION THREAD

**Discussion by hansotto1** [wiresock/WireSockUI#71] [Q&A]
I would like to start WireSockUI as a domain user without admin privileges. But when I simply click on WireSockUI.exe I always have to provide admin credentials...?

**Comment by wiresock** [comment]
If an application detects the possibility to elevate its privileges, it will attempt to do so, but you can decline the elevation. If you prefer the application not to attempt elevation, you can change this default behavior in the Settings.
